pkg/rc/replication_controller.go

```diff
@@ -178,22 +182,20 @@ func New(
 }
 
 func (rc *replicationController) ID() fields.ID {
```

Comment:
Is this function worth keeping around? When you call `rc.rcWatcher.Watch` below you ignore this function and just use `rc.rcID` directly.